packages/playwright-core/src/utils/processLauncher.ts

```diff
@@ -190,14 +190,23 @@ export async function launchProcess(options: LaunchProcessOptions): Promise<Laun
     }
   }
 
+  function sleep(ms: Number) {
+    childProcess.spawnSync(process.execPath, ['-e', 'setTimeout(function(){},' + ms + ')']);
```

Comment:
I can see what you did here. Let's try finding a smaller hammer.

Reply:
Yes, I found a lot of methods, the most stable one is a loop, but it will consume a lot of cpu, so I chose this relatively simple and relatively stable method.